src/App.vue

```diff
@@ -113,6 +138,46 @@ export default {
         )
         .finally(() => (this.status.loading = false));
     }
+  },
+  computed: {
+    layoutBand: function() {
```

Comment:
this is hurting my 4pm brain. My gut feeling is to leave it in but I want to consider whether we really want to be reflowing the app this dynamically. It may be perfect. Not sure!

Reply:
This will most likely not be needed once we get the router in place - but until that lands I won't know for sure. If you're comfortable leaving it in for now, I think that would be best.